@shared/api/types.ts

```diff
@@ -146,7 +147,16 @@ export interface AssetDomains {
   [code: string]: string;
 }
 
-export type Balances = Types.BalanceMap | null;
+export interface TokenBalance extends AssetBalance {
```

Comment:
This new type extends the existing pattern in order to combine the data keys in our store for both classic and soroban assets.